graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsAutoConfiguration.kt

```diff
@@ -289,7 +289,9 @@ open class DgsAutoConfiguration(
             Duration.parse(configProps.preparsedDocumentProvider.cacheValidityDuration),
         )
 
+    // TODO: Remove when legacy modules are removed. This is also handled in DgsSpringGraphQLAutoConfiguration
```

Comment:
nit:  consider adding github issues with any TODO comments to more easily trace your way back to this code